tutorials/shading/your_first_shader/index.rst

```diff
@@ -0,0 +1,26 @@
+Your first shader
```

Comment:
I did not check whether all the new "Your first shader" tutorials are compatible with 3.1.x, CC @clayjohn.

Reply:
Yep! These were written in 3.1 So its all good!